tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -223,9 +225,17 @@ def get_test_shapes(shapes_id: ShapesId):
         ]
     if shapes_id == ShapesId.LARGE:
         return [
-            # some random large sizes
-            TestShape(m=123, k=456, n=789, accumulate=True),
-            TestShape(m=654, k=321, n=234, accumulate=False),
```

Comment:
These look like the only test cases with odd n/k values. Maybe worth keeping or adding something similar?

Reply:
Good point. Added funkification of shapes.